pkg/operator/controller/ingress/load_balancer_service.go

```diff
@@ -468,6 +468,7 @@ var managedLoadBalancerServiceAnnotations = sets.NewString(
 	awsLBHealthCheckIntervalAnnotation,
 	GCPGlobalAccessAnnotation,
 	localWithFallbackAnnotation,
+	awsLBAdditionalResourceTags,
```

Comment:
This will cause the operator to override user changes to the annotation.  @frobware, is there a reason the operator didn't manage this annotation before?  Is documenting the change sufficient, or do we need to backport some logic to set `Upgradeable=False` if the user has set an annotation that doesn't match the tags specified in the infrastructure config?

Reply:
> is there a reason the operator didn't manage this annotation before?

It's not clear to me whether this should have been specified previously or is/was a glaring omission; that's perhaps not terribly helpful but if we go ahead with this change then we should also go with the additional logic (and backports) for smooth upgrades (IMO).

Reply:
@Miciah  @frobware  thanks a lot for your review 

The target of this PR is a watch the AWS resource tags on the `infrastructure` crd and update the tags accordingly. 
Would this change be sufficient for this purpose ? 

cc @arjunrn @lmzuccarelli